src/main/java/seedu/multitasky/model/ModelManager.java

```diff
@@ -271,7 +280,7 @@ private void updateAllFilteredLists(Set<String> keywords, Calendar startDate, Ca
             if (search == Search.POWER_AND || search == Search.POWER_OR) {
                 for (int level = PowerMatch.MIN_LEVEL; level <= PowerMatch.MAX_LEVEL; ++level) {
                     qualifier = new NameDateStateQualifier(keywords, startDate, endDate, states, search,
-                                                           level);
+                            level);
                     updateFilteredEventList(new PredicateExpression(qualifier));
```

Comment:
The previous indentation seems to be better. Is it because your Eclipse is reverted to the default settings?